src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -170,67 +186,72 @@ k6 run scripts/website.js \
   --no-summary
 ```
 
-## Test script optimizations
-
-If you are trying to squeeze more performance out of the hardware, you can consider optimizing the code of the test script itself.
+## Script optimizations
 
-### Checks and groups
+To squeeze more performance out of the hardware, consider optimizing the code of the test script itself.
+Some of these optimizations involve limiting how you use certain k6 features.
+Other optimizations are general to all JavaScript programming.
 
-k6 records the result of every individual check and group separately. If you are using many checks and groups, you may consider removing them to boost performance.
+### Limit resource-intensive k6 operations
 
-### Custom metrics
+Some features of the k6 API entail more computation to execute.
+To maximize load generation, you might need to limit how your script uses the following.
 
-Similar to checks, values for custom metrics (Trend, Counter, Gauge and Rate) are recorded separately. Consider minimizing the usage of custom metrics.
+<DescriptionList>
 
-### Thresholds with abortOnFail
+Checks and groups
+: k6 records the result of every individual check and group separately. If you are using many checks and groups, you may consider removing them to boost performance.
 
-If you have configured [abortOnFail thresholds](/using-k6/thresholds#aborting-a-test-when-a-threshold-is-crossed), k6 needs to evaluate the result constantly to verify that the threshold wasn't crossed. Consider removing this setting.
+Custom metrics
+: Similar to checks, values for custom metrics (Trend, Counter, Gauge and Rate) are recorded separately. Consider minimizing the usage of custom metrics.
 
-### URL grouping
+Thresholds with abortOnFail
```

Comment:
Should `abortOnFail` be formatted as code here, same as `discardResponseBodies`? Or is this not possible currently because of #1002?

Reply:
Exactly. I'd prefer to format this and the error headings as titles, but those are the breaks.